### DISCUSSION TITLE
Should the `postbuild` command added in the vercel's build command?

### DISCUSSION THREAD

**Discussion by skwowet** [iamvishnusankar/next-sitemap#558] [Q&A] [answered]
I have set up the **next-sitemap** for my NextJS project, but confused how it works 😕 

Once after I run the `postbuild` command locally after `build` it generates sitemap and stuff in the `/public` directory, is that enough? Or should I have to add the `postbuild` with vercel's build command?

P.S I really like the project, appreciate your effort in maintaining it. Also, the instruction missed out the above information whether the `postbuild` should be added with deployment platform or not, maybe I could open up a PR so that it helps people coming up with the very same confusion.

**Comment by iamvishnusankar** [comment, marked as answer]
@yeganathan18 You are not required to add "postbuild" script on vercel. The CLI will automatically run the post-script hook. Read more about it here: https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts

**Reply by skwowet** [reply]
Thanks for the reply, haven't really looked upon that before 👍🏻
